File: JAVA/Person.java
// Kelas dasar Person -> akan diwariskan ke Student, Lecturer, Staff
public class Person {
    private String id;
    private String nama;
    private int usia;

    // Constructor default
    public Person() {
        this.id = "";
        this.nama = "";
        this.usia = 0;
    }

    // Constructor dengan parameter
    public Person(String id, String nama, int usia) {
        this.id = id;
        this.nama = nama;
        this.usia = usia;
    }

    // Getter dan Setter
    public String getId() { return id; }
    public void setId(String id) { this.id = id; }

    public String getNama() { return nama; }
    public void setNama(String nama) { this.nama = nama; }

    public int getUsia() { return usia; }
    public void setUsia(int usia) { this.usia = usia; }
}

File: PYTHON/person.py
# person.py
# Kelas dasar Person -> akan diwariskan ke Student, Lecturer, Staff

class Person:
    def __init__(self, id="", nama="", usia=0):
        self.__id = id
        self.__nama = nama
        self.__usia = usia

    # Getter
    def get_id(self): return self.__id
    def get_nama(self): return self.__nama
    def get_usia(self): return self.__usia

    # Setter
    def set_id(self, id): self.__id = id
    def set_nama(self, nama): self.__nama = nama
    def set_usia(self, usia): self.__usia = usia